pages/ecosystem.vue

```diff
@@ -243,13 +271,22 @@ const joinAction: Link = {
     margin-top: carbon.$spacing-07;
   }
 
-  &__search {
+  &__toolbar {
+    margin-top: carbon.$spacing-06;
+  }
+
+  &__search,
```

Comment:
Since we'll add the label-text, we can remove the `&__sort-dropdown` from needing a `margin-top`. We can leave it on the `&__search` class so it balances out... 

```scss
&__search {
  margin-top: carbon.$spacing-06;

  --cds-field-01: #{carbon.$cool-gray-10};
  --cds-field-04: #{carbon.$cool-gray-30};
}
```

Reply:
Resolved in https://github.com/Qiskit/qiskit.org/pull/3286/commits/186ea075d560882cfac850ff1f0ef5633910d571